ubuntu-advantage.1

```diff
@@ -75,6 +75,48 @@ Possible values are: active, inactive, or n/a
 Show version of the Ubuntu Advantage package.
 
 
+.SH CONFIGURATION
+By default, ubuntu-advantage client configuration options are read from
+(/etc/ubuntu-advantage/uaclient.conf). If a (uaclient.conf) is present in
+the current working directory it will be used instead of the default conf file.
+
+The following configuration options are available:
+.TP
+.B
```

Comment:
can you sort these the same as the config file itself?

I show:

- sso_auth_file
- contract_url
- data_dir
- log_level
- log_file